code/game/objects/items/rogueweapons/intents.dm

```diff
@@ -546,6 +546,39 @@
 	canparry = TRUE
 	item_d_type = "blunt"
 
+// Generic blunt "strike" for creatures that shouldn't cut or stab
+/datum/intent/simple/strike
+	name = "strike"
+	icon_state = "instrike"
+	attack_verb = list("strikes", "slams")
+	animname = "blank22"
+	blade_class = BCLASS_BLUNT
+	hitsound = "punch_hard"
+	chargetime = 0
+	penfactor = 10
+	swingdelay = 0
+	candodge = TRUE
+	canparry = TRUE
+	item_d_type = "blunt"
```

Comment:
this is unused and should probably be removed, `jitterskull_crush` isn't even a subtype of it. also notably, `/datum/intent/simple/slam` already exists anyway